Alert/Alert.css

```diff
@@ -0,0 +1,134 @@
+@import "../../utility/component.css";
+
+.alert{
+    position: relative;
+    padding: 1rem 1rem;
+    margin-bottom: 1rem;
+    border: 1px solid transparent;
+    border-radius: 0.25rem;
+}
+
+/*---alert example---*/
+
+#show_alert{
```

Comment:
As Vikram also mentioned once. Consistency not only in UI but in everything even in our code is a must. Use either classes or IDs.

Reply:
It's just to change the priority of the "live example" alert component. I will update my code.

Reply:
changed ID -> Class